smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/JaegerExporterSmokeTest.groovy

```diff
@@ -21,8 +21,8 @@ class JaegerExporterSmokeTest extends SmokeTest {
   @Override
   protected Map<String, String> getExtraEnv() {
     return [
-      "OTEL_TRACES_EXPORTER"          : "jaeger",
-      "OTEL_EXPORTER_JAEGER_ENDPOINT" : "collector:14250"
+            "OTEL_TRACES_EXPORTER"          : "jaeger",
+            "OTEL_EXPORTER_JAEGER_ENDPOINT" : "http://collector:14250"
```

Comment:
Could we remove this test? 

We don't officially support Jaeger exporter.  We support only OTLP and ZIPKIN

Reply:
done